### ISSUE TITLE
Flashing 

### ISSUE THREAD

**Issue by 17dshaikh** [spleenware/ripple#53]
Does it matter when I am flashing it whether it is a 433mhz or a 900mhz one by the way I am using a feather

**Comment by spleenware** [comment]
No, there is just a single firmware now, but you set the frequency in
the Android apps.
